crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1072,41 +979,6 @@ pub fn prepare_mesh_bind_group(
     }
 }
 
-// NOTE: This is using BufferVec because it is using a trick to allow a fixed-size array
-// in a uniform buffer to be used like a variable-sized array by only writing the valid data
-// into the buffer, knowing the number of valid items starting from the dynamic offset, and
-// ignoring the rest, whether they're valid for other dynamic offsets or not. This trick may
-// be supported later in encase, and then we should make use of it.
```

Comment:
this comment disappeared, is it no longer relevant?

Reply:
I just didn't understand the comment. I thought it was standard usage of `BufferVec`. I used the same method when implementing morph targets. The comment comes from #4339

Reply:
I asked `@superdump` about it and copied their answer to replace this comment. I also added a reference to the comment in the `morph::extract_morphs` system.